selfdrive/controls/lib/lane_planner.py

```diff
@@ -24,7 +24,7 @@ def calc_d_poly(l_poly, r_poly, p_poly, l_prob, r_prob, lane_width):
 
 
 class LanePlanner(object):
-  def __init__(self):
+  def __init__(self,withAlca):
```

Comment:
Consider prepending the parameter such that it's clear it's a boolean. Example `isAlcaEnabled` `shouldUseAlca`

Reply:
done